### PR TITLE
CSS logo

### LINKED ISSUE #567: i18n - Reimplement Almanac logo using text and CSS instead of SVG
The [PNG Logo](https://almanac.httparchive.org/static/images/logo.png) contains English wording. It looks like it's created from [the SVG logo](https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/static/images/logo.svg) unfortunately, the SVG logo has been optimized for Performance and all original text has been replaced by paths.

Is there a way to get an editable version to localize the "By" wording (in French, "par").

### PR DESCRIPTION
**Pull request by catalinred**
Fixes #567
The logo is now made with CSS

- The logo degrades nicely when switching to an `rtl` text direction.
- Adress the Japan edge case when the proper logo translation is a bit different than just the preposition text. 
  Looking forward to @MSakamaki feedback here.
- The logo is perfectly scalable by adjusting `.navigation-logo`'s font-size size.
- I also fixed a horizontal scroll issue that I've noticed recently on the index page.

Let me know your thoughts!

### CODE REVIEW

**Comment by borisschapira** [comment]
Ok for the French modifications 👍

**Comment by bazzadp** [comment]
This sounds awesome @catalinred. Will check it out later.

File: src/templates/base/2019/base.html [hunk 1]
```diff
@@ -104,7 +107,19 @@
...
+            <span class="wa">{{ self.web_almanac() }}</span>
+            <span class="line-group">  
+              <span class="pre">{{ self.preposition() }}</span>
+              <span class="ha">{{ self.http_archive() }}</span>
+            </span>
+          {% else %}
+            <span class="ha line-group">{{ self.http_archive() }}</span>            
+            <span class="pre">{{ self.preposition() }}</span>
+            <span class="wa">{{ self.web_almanac() }}</span>                        
+          {% endif %}
```
Comment (bazzadp) [inline src/templates/base/2019/base.html:121]: This is very clever, but not loving the hard coding of the language.

What would you think about moving this to the templates? So replacing this with just:

```suggestion
          {{ self.web_almanac_logo() }}
```

And then having the following in `en/2019/base.html`:

```
{% block web_almanac_logo %}
<span class="wa">Web Almanac</span>
<span class="line-group">  
   <span class="pre">By</span>
   <span class="ha">HTTP Archive</span>
</span>
{% endblock %}
```

Yes it does put a bit of code/presentation logic into the language templates but we have a few other examples of this, gives us much greater flexibility for other languages in the future, and avoid repeating this code three times.

File: src/templates/base/2019/base_ebook.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
   @page {
     @top-left { content: ''; }
     @top-right { content: string(chapter-subtitle) string(chapter-title); }
-    @bottom-left { content: "{{ year }} {{ self.organization() }}"; }
+    @bottom-left { content: "{{ year }} {{ self.web_almanac() }} {{ self.preposition() }} {{ self.http_archive() }}"; }
```
Comment (bazzadp) [inline src/templates/base/2019/base_ebook.html:31]: This doesn't work for Japanese. So think we should maybe keep `self.organization()` (and the associated block).

File: src/templates/base/2019/base_ebook.html [hunk 3]
```diff
@@ -45,7 +45,7 @@
     @top-left { content: string(chapter-subtitle) string(chapter-title); }
     @top-right { content: ''; }
     @bottom-left { content: counter(page); }
-    @bottom-right { content: "{{ year }} {{ self.organization() }}"; }
+    @bottom-right { content: "{{ year }} {{ self.web_almanac() }} {{ self.preposition() }} {{ self.http_archive() }}"; }
```
Comment (bazzadp) [inline src/templates/base/2019/base_ebook.html:48]: Same here

File: src/templates/base/2019/base_ebook.html [hunk 4]
```diff
@@ -54,7 +54,7 @@
   @page :right {
     @top-left { content: ''; }
     @top-right { content: string(chapter-subtitle) string(chapter-title); }
-    @bottom-left { content: "{{ year }} {{ self.organization() }}"; }
+    @bottom-left { content: "{{ year }} {{ self.web_almanac() }} {{ self.preposition() }} {{ self.http_archive() }}"; }
```
Comment (bazzadp) [inline src/templates/base/2019/base_ebook.html:57]: Same here

File: src/templates/base/2019/base.html (hunk 1 above)
Comment (catalinred) [reply src/templates/base/2019/base.html]: I did update this as requested! ;)

File: src/templates/base/2019/base_ebook.html (hunk 2 above)
Comment (catalinred) [reply src/templates/base/2019/base_ebook.html]: Done.

File: src/templates/base/2019/base_ebook.html (hunk 3 above)
Comment (catalinred) [reply src/templates/base/2019/base_ebook.html]: Done.

File: src/templates/base/2019/base_ebook.html (hunk 4 above)
Comment (catalinred) [reply src/templates/base/2019/base_ebook.html]: Done.

**Comment by catalinred** [comment]
Awesome, thanks for merging.

**Pull request merged**